packages/widgets/src/components/SearchBox/SearchBox.tsx

```diff
@@ -0,0 +1,70 @@
+import { Icon } from '@ssa-ui-kit/core';
+import { SearchBoxWrapper } from './SearchBoxWrapper';
```

Comment:
Let's import 3rd party modules 1st and then the internal ones. Other files follow this convention.